config/production.env

```diff
@@ -0,0 +1,13 @@
+LOG_LEVEL=debug
+LOG_LEVEL=info
+DB_HOST=10.146.200.8
+DB_PORT='1032'
+DB_PSWD=AQECAHh7ea2tyZ6phZgT4B9BDKwguhlFtRC6hgt+7HbmeFsrsgAAAG4wbAYJKoZIhvcNAQcGoF8wXQIBADBYBgkqhkiG9w0BBwEwHgYJYIZIAWUDBAEuMBEEDPIaMdJ70N4arvdlfgIBEIAr2OTX22QsmU1LpexxEDrfxjtnFqcm4pCnfDoP/z80c8iW2DEtvIOrsY7o0A==
+DB_USER=AQECAHh7ea2tyZ6phZgT4B9BDKwguhlFtRC6hgt+7HbmeFsrsgAAAGcwZQYJKoZIhvcNAQcGoFgwVgIBADBRBgkqhkiG9w0BBwEwHgYJYIZIAWUDBAEuMBEEDNapmZKmWxO59lrUhAIBEIAkPcsXj35Is8+naFzWIsWYMa9Yvt+aJZqfFZgDicQSnmEUAWBr
+DB_NAME=iii
+DB_QUERY='SELECT sierra_view.holding_record_card.id, sierra_view.holding_record_card.holding_record_id,
```

Comment:
Why is this an env var? A few issues:
 - It looks like there's a typo in the production query: "sierra_view.hold_view" should be "sierra_view.holding_view"?
 - After above change, I believe the qa and prod versions are identical, so unclear why they would need to be separately configured.
 - It feels dangerous to pass whole queries directly to the db from ENV. Someone with no understanding of the app but access to app config could easily construct a harmful query. SQL queries should pass through code review.
 - Also, I don't think most `.env` parsing utils will parse this multi-line value the way you want. (I think you'd need to escape line breaks.)

Reply:
Thanks for catching that typo. This approach of passing the SQL query as an ENV variable was adapted from the poller. I don't have strong feelings here but maybe @mwbenowitz can weigh in if there is an advantage to using the ENV variable approach, otherwise I can move it.

Reply:
This was set as an ENV with the idea that this could be a general (or be adapted to be a general) poller for queries that had to be run directly in the iii database (in the same way that other pollers work for bib/item/holding records.

That idea wasn't carried through here though, and with this becoming the check-in card service it's probably out of date. And that's assuming this was a good idea in the first place. It would definitely be safe to move this into the code and probably easier to understand.

Reply:
OK I've moved this into the code.